src/reachy2_sdk/reachy_sdk.py

```diff
@@ -113,6 +113,10 @@ def connect(self) -> None:
         self._sync_thread.daemon = True
         self._sync_thread.start()
 
+        self._audit_thread = threading.Thread(target=self._audit)
+        self._audit_thread.daemon = True
```

Comment:
We can directly set daemon=True in the line above. (Same for the other thread above)